app/mailers/discovered_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class DiscoveredMailer < ::ApplicationMailer
+  helper :discovered_hosts
+  def discovered_summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user = User.find(options[:user])) && user.mail_enabled?
+    set_url
+    @time = options[:time] || 'yesterday'
```

Comment:
It shows as "Summary from yesterday ago to now" in the report. Maybe we can do "1 day" here instead.